docs/release-notes/latest.md

```diff
@@ -9,100 +9,100 @@ tags:
-  * [<i class="fas fa-external-link fa-xs"></i> 1.41.0](https://github.com/returntocorp/semgrep/releases/tag/v1.41.0)
-  * [<i class="fas fa-external-link fa-xs"></i> 1.42.0](https://github.com/returntocorp/semgrep/releases/tag/v1.42.0)
+- The following versions of Semgrep OSS Engine were released in October 2023:
+  - [<i class="fas fa-external-link fa-xs"></i> 1.43.0](https://github.com/returntocorp/semgrep/releases/tag/v1.43.0)
+  - [<i class="fas fa-external-link fa-xs"></i> 1.44.0](https://github.com/returntocorp/semgrep/releases/tag/v1.44.0)
+  - [<i class="fas fa-external-link fa-xs"></i> 1.45.0](https://github.com/returntocorp/semgrep/releases/tag/v1.45.0)
+  - [<i class="fas fa-external-link fa-xs"></i> 1.46.0](https://github.com/returntocorp/semgrep/releases/tag/v1.46.0)
 
 ## 🌐 Semgrep Cloud Platform
 
 ### Added
+- Added a button to **Remove** source code manager (SCM) apps. This is helpful when you may have a malconfigured SCM app, such as GitHub's `semgrep-app`, and want to reinstall it. <!--(10688)--> To remove an SCM, click **<i class="fa-solid fa-gear"></i> Settings > Source code managers**.
+![Remove your source code manager](/img/settings-scm-remove.png)
+- Added Semgrep Assistant to the new onboarding flow. <!--(10716) -->
+- **OpenAPI:** Renamed instances of r2c to Semgrep. <!--(10685) -->
+- **Projects page:** Added a new column to display a Semgrep Secrets counter. This counter counts all secrets regardless of validation state. <!--(10588)-->
+- **CLI login:** New users are now directed to create a Semgrep org when they are logging in for the first time to Semgrep Cloud Platform from the CLI. <!-- (10596) -->
+
+### Fixed
 
-- UX: Added a new **onboarding** flow. This onboarding flow streamlines the following steps to ensure that users are able to quickly add repositories for scanning with Semgrep: <!-- #10473 -->
-	- **Deployment creation**. The Semgrep team has made improvements to Semgrep account creation and connecting your source code manager, such as GitHub or GitLab. 
-	- **Onboarding checklist.** This helps you troubleshoot and resolve any issues early on in your journey.
-	- **Tour of features**. Make the most of your Semgrep experience by learning what features are available to you.
-- Logging into Semgrep Cloud Platform through the CLI associates your CLI user ID to your Semgrep Cloud Platform account. See the [<i class="fas fa-external-link fa-xs"></i> Anonymous User ID](https://github.com/returntocorp/semgrep/blob/develop/PRIVACY.md#anonymous-user-id) section for more details.
+- Fixed UI issues in the new onboarding flow.
+- Fixed an issue where Semgrep Cloud Platform could crash during the onboarding flow. <!--(#10940) -->
+- Various frontend fixes and improvements to the following:
+	- Finding detail page
+	- Projects page
+- Fixed an issue where the **Delete user** functionality did not work for some Semgrep orgs. <!-- (#10756) -->
+- Corrected SARIF output by adding back a missing dict item.
```

Comment:
I'm not sure what this means?

Reply:
There was a missing mapping from Info to Note: [https://github.com/semgrep/semgrep/pull/9077](https://github.com/semgrep/semgrep/pull/9077)

Reply:
In hindsight - this is actually an OSS fix so I'll remove it.

Reply:
Ah great!